hdf5-vol-daos.spec

```
@@ -197,6 +197,9 @@ done
 %endif
 
 %changelog
+* Mon Dec 14 2020 Kenneth Cain <kenneth.c.cain@intel.com> - 0.1-4.gb8e6afb18f
```

Hash in the version string needs updating:
```suggestion
* Mon Dec 14 2020 Kenneth Cain <kenneth.c.cain@intel.com> - 0.1-4.gfcbdc0b
```